src/main/java/io/codelex/flightplanner/flightmanager/service/FlightService.java

```diff
@@ -0,0 +1,35 @@
+package io.codelex.flightplanner.flightmanager.service;
+
+import io.codelex.flightplanner.flightmanager.domain.Airport;
+import io.codelex.flightplanner.flightmanager.domain.Flight;
+import io.codelex.flightplanner.flightmanager.dto.PageResult;
+import io.codelex.flightplanner.flightmanager.dto.SearchFlightRequest;
+
+import java.util.List;
+
+public interface FlightService {
+    void clearFlights();
+
+    void addFlight(Flight req);
+
+    void deleteFlight(Long id);
+
+    default boolean isValidAirports(Flight req) {
```

Comment:
Es domāju šitās divas metodes, kuras uztaisīji kā default labāk būtu likt citā klasē. Vari uztaisīt AbstractFlightService un viņā ieliekt abas un gan memory, gan db service var extendot to AbstractFlightService.

Reply:
Ok, tiks darīts. Interfeisos likt default metodes ir slikta prakse, vai tam ir kāds cits iemesls, ka jātaisa atsevišķa abstrakta klase?